app/models/manga.rb

```diff
@@ -31,82 +31,23 @@
 #
 
 class Manga < ActiveRecord::Base
-  include Versionable
-  include BayesianAverageable
+  PG_TITLE_SCOPE = %i(romaji_title english_title)
 
-  include PgSearch
-  pg_search_scope :instant_search,
-    against: [ :romaji_title, :english_title ],
-    using: { tsearch: { normalization: 42, dictionary: 'english' } },
-    ranked_by: ':tsearch'
-  pg_search_scope :full_search,
-    against: [ :romaji_title, :english_title ],
-    using: {
-      tsearch: { normalization: 42, dictionary: 'english' },
-      trigram: { threshold: 0.1 }
-    },
-    ranked_by: ':tsearch + :trigram'
-
-  extend FriendlyId
-  friendly_id :romaji_title, use: [:slugged, :history]
+  VALID_TYPES =  ["Manga", "Novel", "One Shot", "Doujin", "Manhwa", "Manhua", "OEL"]
```

Comment:
I went over this PR like 4 times trying to find any problems (was mostly thinking with `self`, since we had a couple of those mess up in a previous refactor) but the only thing I was able to come up with was that these should be single quotes since they don't have interpolation.  Everything else looks fine.

Fix 'em and LGTM :+1: